source/plugins/go/src/extension/extension.go

```diff
@@ -35,38 +36,33 @@ func GetInstance(flbLogger *log.Logger, containertype string) *Extension {
 	return singleton
 }
 
-func getExtensionConfigs() ([]ExtensionConfig, error) {
+func getExtensionData() (TaggedData, error) {
 	guid := uuid.New()
 	taggedData := map[string]interface{}{"Request": "AgentTaggedData", "RequestId": guid.String(), "Tag": "ContainerInsights", "Version": "1"}
 	jsonBytes, err := json.Marshal(taggedData)
 
-	var data []byte
-	enc := codec.NewEncoderBytes(&data, new(codec.MsgpackHandle))
-	if err := enc.Encode(string(jsonBytes)); err != nil {
-		return nil, err
-	}
-
-	fs := &FluentSocket{}
-	fs.sockAddress = "/var/run/mdsd-ci/default_fluent.socket"
-	if containerType != "" && strings.Compare(strings.ToLower(containerType), "prometheussidecar") == 0 {
-		fs.sockAddress = fmt.Sprintf("/var/run/mdsd-%s/default_fluent.socket", containerType)
-	}
-	
-	responseBytes, err := FluentSocketWriter.writeAndRead(fs, data)
-	defer FluentSocketWriter.disconnect(fs)
+	responseBytes, err := getExtensionConfigResponse(jsonBytes)

```

Comment:
getExtensionConfigResponse is linux specific code, how are you preventing this code not getting triggered for windows?

Reply:
There is both linux and windows version of the function. Have added two files.